app/src/main/java/org/oppia/android/app/utility/datetime/DateTimeUtil.kt

```diff
@@ -31,3 +34,54 @@ class DateTimeUtil @Inject constructor(
     fun getDateTimeUtil(): DateTimeUtil
   }
 }
+
+const val SECOND = 1
+const val MINUTE = 60 * SECOND
+const val HOUR = 60 * MINUTE
+const val DAY = 24 * HOUR
+const val MONTH = 30 * DAY
+const val YEAR = 12 * MONTH
+
+private fun currentDate() = Calendar.getInstance().timeInMillis
+
+/**
+* Returns the readable string of the duration from the provided time in [Long].
+*/
+fun Long.timeAgo(view: View): String {
+  val time = this
+  val now = currentDate()
+  val diff = (now - time) / 1000
+
+  val resourceHandler = getResourceHandler(view)
+
+  return when {
+    diff < MINUTE -> resourceHandler.getStringInLocale(R.string.just_now)
```

Comment:
I think fixing this string actually involves introducing an Android quantity string with formatting that combines all of the different cases into one. This seems like the ideal use case for quantity strings as it will provide a full context to translators such that they can specialize the wording to better fit different languages.